src/probnum/filtsmooth/gaussfiltsmooth/iterated_kalman.py

```diff
@@ -71,7 +77,11 @@ def filter_step(
             linearise_predict_at = previous_posterior(start)
             linearise_update_at = previous_posterior(stop)
         pred_rv, info_pred = self.predict(
-            start, stop, current_rv, linearise_at=linearise_predict_at
+            start,
+            stop,
+            current_rv,
+            linearise_at=linearise_predict_at,
+            intermediate_step=intermediate_step,
```

Comment:
Is this just something that was overlooked in the IEKS PR?

Reply:
not sure what you mean, but if I remember, I added something in the IEKS things that were missing before. Could have been this input.